src/quo/components/list_items/network_list_item/style.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo.components.list-items.network-list-item.style
+  (:require [quo.foundations.colors :as colors]))
+
+(defn- background-color
+  [state customization-color theme]
+  (cond
+    (= state :pressed)  (colors/resolve-color customization-color theme 5)
+    (= state :active)   (colors/resolve-color customization-color theme 10)
+    (= state :selected) (colors/resolve-color customization-color theme 5)
```

Comment:
I think we can remove `selected` as it's not an option in Figma 👍

Reply:
remove 👍